punx/validations/hdf5_group_items_in_base_class.py

```diff
@@ -112,7 +112,7 @@ def verify_group_attributes(validator, v_item, base_class):
             test_name = "value of @" + k
             status = finding.TODO
             c = "TODO: need to validate"
-            c += ": @" + k + " = " + v
+            c += ": @" + k + " = " + str(v)
```

Comment:
Nice.  Since: https://github.com/prjemian/punx/blob/32d9c3b1aee189367a33bdf34c2f1a6a2b9ccc43/.github/workflows/unit-tests-pytest.yml#L11

we could use an f-string here

Reply:
yes I was tempted but there are many similar lines around there which we could use an f-string.

Reply:
Rabbit hole alert!  😁